### PR TITLE
Just some little suggestions

### PR DESCRIPTION
**Pull request by kiuby88**
Just some little suggestions

### CODE REVIEW

**Comment by kiuby88** [comment]
@andreaturli  just a couple of suggestions. Feel free for declining or merging

File: brooklyn-cloudfoundry/src/main/java/org/apache/brooklyn/cloudfoundry/entity/CloudFoundryAppFromManifestImpl.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ private void checkConfiguration() {
         String configurationContents = getConfig(CONFIGURATION_CONTENTS);
 
         // Exactly one of the two must have a value
-        if (Strings.isBlank(configurationUrl) == Strings.isBlank(configurationContents))
+        if (Strings.isBlank(configurationUrl) && Strings.isBlank(configurationContents))
```
Comment (andreaturli) [inline brooklyn-cloudfoundry/src/main/java/org/apache/brooklyn/cloudfoundry/entity/CloudFoundryAppFromManifestImpl.java]: I think with `&&` you want `!Strings` instead

Comment (kiuby88) [reply brooklyn-cloudfoundry/src/main/java/org/apache/brooklyn/cloudfoundry/entity/CloudFoundryAppFromManifestImpl.java]: You are right. I did not understand well the Exactly one condition :-) I have reverted this kind of changes. Sorry

**Comment by andreaturli** [comment]
Thanks @kiuby88 lgtm

**Pull request closed without merging**